.pre-commit-config.yaml

```
@@ -1,3 +1,11 @@
+default_install_hook_types:
+  - pre-commit
+  - commit-msg
+  - post-checkout
```

```suggestion
  - pre-push
```
还是改回pre-push好了，推的时候触发的确有必要
我们要修改documentation的CI,额外加一个test用--hook-stage来显式调用这个hook，默认的stage是commit